### ISSUE TITLE
Can DROID be integrated into a records management system?

### ISSUE THREAD

**Issue by kathaurielle** [digital-preservation/droid#943]
Hello, a user is building a specific Record Management System for public institutions, any they have to check files on input, including type of file. Can they intergrate PRONOM/ DROID to so this?

**Comment by steve-daly** [comment]
Yes, there are various ways to integrate with DROID, for example calling the command line variant to return details on each file/folder scanned.  I'd encourage the user to to interact directly on this thread to discuss it further.
